skcosmo/datasets/_base.py

```diff
@@ -0,0 +1,32 @@
+from os.path import join, dirname
+import numpy as np
+from sklearn.utils import Bunch
+
+
+def load_degenerate_manifold():
```

Comment:
how about `load_dataset(name)`, that we can then use as `load_dataset("degenerated_CH4_manifold")`?

Reply:
Ok, so this is following sklearn convention, so I guess it can stay as is. Maybe rename it to `load_degenerate_CH4_manifold`, `load_degenerate_methane_manifold` or just `load_degenerate_methane`?

Reply:
renamed